templates/base.html

```diff
@@ -120,7 +120,13 @@ <h1><a href="{{ get_url(path="/", trailing_slash=true) }}">{{ config.extra.textl
   <hr>
   <center>
     <p>{{ social_macros::social_links(social_config=config.extra.social) }}</p>
-    {# Copyright #}<p>Copyright &copy; 2019-{{ now() | date(format="%Y") }} {{ config.title }}</p>
+
+    {# Copyright #}
+    {% set start_year=config.extra.start_year | default(value="") %}
+    {% if start_year %}{% set start_year=start_year ~ "-" %}{% endif %}
```

Comment:
note that I could have combined this line and line 125, but doing so would cause zola to evaluate config.extra.start_year and throw an error if it didn't exist in the config.toml. by separating them like this, we can support the option if it's unset, but allow existing configs to work fine without modification (though they'll need to specify a start_year if the fullcopyright range is needed on the page... at least they won't break with this change though... is my point 😄 )